.gitignore

```diff
@@ -24,6 +24,9 @@ dotnet-uninstall-debian-packages.sh
 # Visual Studio IDE directory
 .vs/
 
+# VS Code directory
+/**/.vscode/
```

Comment:
In home folder we have `.vscode` that should remain under git control.

Reply:
This is to ignore `.vscode` folders in the sub directories. Changes to the one in the root directory is not ignored.